src/TwigComponent/src/Command/TwigComponentDebugCommand.php

```diff
@@ -182,12 +182,12 @@ private function findAnonymousComponents(): array
             ->name('*.html.twig')
         ;
         foreach ($finderTemplates as $template) {
-            $component = str_replace('/', ':', $template->getRelativePathname());
+            $component = str_replace(\DIRECTORY_SEPARATOR, ':', $template->getRelativePathname());
```

Comment:
What about the other "/" in this file ? 

Especially the "notPath" ?

Maybe we should use Filesystem / Path to ensure all is still working as expected ?